src/api/go/pxapi/client.go

```diff
@@ -88,6 +91,10 @@ func NewClient(ctx context.Context, opts ...ClientOption) (*Client, error) {
 func (c *Client) init(ctx context.Context) error {
 	isInternal := strings.Contains(c.cloudAddr, "cluster.local")
 
+	if tlsDisabled := os.Getenv("PX_DISABLE_TLS"); tlsDisabled != "1" && isInternal {
```

Comment:
Should this be:
```
tlsDisabled := os.Getenv("PX_DISABLE_TLS") == "1";
insecureSkipVerify := tlsDisabled && isInternal;
```
We can add a helpful log message/hint to the eventual TLS failure later. Also maybe this should be a cmd line arg. like `--disable-tls`?

Reply:
Based on our in person conversation, I've updated this to reflect the above.